### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by StillLutto**
Here are a couple of styling changes for beauty and consistency.
> Please be as critical as possible as this is a highly subjective topic!

## Features
- Better scoreboard
- Better looking spawn text displays
- Improve tab list
- Better MOTD
- Make the +1 second message in KOTH display in the actionbar instead of a default message
- Convert almost all normal text into small text
- More consistent colors
- Add donations.disabled config

## Code Changes/Additions
- Add Colors enum
- Add TagResolvers for all of the colors
- Convert DonateEvent from an Event to a TwilightEvent
- Some slight code refactoring

### CODE REVIEW

File: src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt [hunk 1]
```diff
@@ -43,7 +45,7 @@ class ChristmasEventPlugin : JavaPlugin() {
     var canJoin = false
 
     companion object {
-        val instance: ChristmasEventPlugin by lazy { getPlugin(ChristmasEventPlugin::class.java) }
+        val INSTANCE: ChristmasEventPlugin by lazy { getPlugin(ChristmasEventPlugin::class.java) }
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: Technically, a `by lazy` property is not a compile-time constant, so even if the variable is to emphasise its role as a global, immutable, always-present reference, idiomatically it is correct to use camelCase.

**Comment by ShreyasAyyengar** [comment]
Are you able to provide screenshots of the colours that you have changed? No need to show text changes, but I'd like public access to visualise the colour scheme.

**Review by ShreyasAyyengar — CHANGES_REQUESTED** [review]
Before I continue to review this, I would like the variable casing to be changed as this significantly decreases the amount of files that are required to review.

**Comment by StillLutto** [comment]
> Are you able to provide screenshots of the colours that you have changed? No need to show text changes, but I'd like public access to visualise the colour scheme.

@ShreyasAyyengar Here is a color palette I quickly made https://artincontext.org/colorpalette/index.html?colors=F23A3A-FF9800-F19C79-E89EB8-EE57FF-B119C2-6735CE-384B70-226F54-A7C957-FFEE63-F3F7F0-42D9C8

File: src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt [hunk 2]
```diff
@@ -56,7 +56,7 @@ class ChristmasEventPlugin : JavaPlugin() {
         registerCommands()
         registerEvents()
         registerPacketAPI()
-        handleDonations()
+        if (!config.getBoolean("donations.disabled")) handleDonations()
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: This is pretty unnecessary to add, as the method `handleDonations()` is intentionally disabled as we do no have the credentials yet. I would remove it entirely from the config and in the code. There isn't really a situation where this would be reasonably disabled.

File: src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt [hunk 3]
```diff
@@ -62,35 +63,36 @@ class HousekeepingEventListener : Listener, PacketListener {
...
-            ).forEach {
-                this.listedPlayers.add(PaperServerListPingEvent.ListedPlayerInfo(it.toLegacyString().colourise(), UUID.randomUUID()))
+            val serverListPingText = listOf(
+                "<st><grey>        <reset> ❆ <bold><light_purple>ꜰʟʏᴛᴇ.ɢɢ <red>ᴄʜʀɪsᴛᴍᴀs ᴇᴠᴇɴᴛ <reset><white>❆ <reset><st><grey>         ",
+                "  <bold><light_purple>ꜰʟʏᴛᴇ.ɢɢ <grey>• <yellow>ᴄᴀʀʙᴏɴ.ʜᴏꜱᴛ <grey>• <blue>ʙᴜɪʟᴛʙʏʙɪᴛ.ᴄᴏᴍ",
+                "",
+                "            <gold>ᴊᴏɪɴ <green><b>ɴᴏᴡ <reset><gold>ᴛᴏ ᴘʟᴀʏ <aqua>x-ᴍᴀѕ ᴍɪɴɪɢᴀᴍᴇѕ",
+                "              <gold>ᴀɴᴅ ѕᴜᴘᴘᴏʀᴛ <red>[ᴄʜᴀʀɪᴛʏ ɴᴀᴍᴇ]" // TODO fill charity name and centre
+            )
+            serverListPingText.map { MiniMessage.miniMessage().deserialize(it) }.forEach {
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt:93]: I have a feeling (and I will test it later) that this will produce the unintended effect of unrendered colours to the client. Player lists do not support components, and the tag resolvers parse custom colours, which will result in the colours breaking and/or being randomised.

File: src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt [hunk 4]
```diff
@@ -273,8 +271,12 @@ class EventController() {
...
      */
     fun updateDonationBar() {
-        donationBossBar.name("<colour:#1DF1BC>ᴅᴏɴᴀᴛɪᴏɴ ɢᴏᴀʟ: <white>$<green>${totalDonations}/${donationGoal}".style())
-        var progress = (totalDonations.toFloat() / donationGoal)
+        donationBossBar.name(getBossBarMessage())
+        val progress = (totalDonations.toFloat() / donationGoal)
         donationBossBar.progress(progress)
     }
+
+    private fun getBossBarMessage(): Component =
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt:285]: Remove explicit return-type declaration

File: src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package gg.flyte.christmas.util
+
+fun String.toSmallText(): String = lowercase()
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt]: I am not in favour of such a util method to be honest. I initially had plans to get this implemented, but the issue arises with users who have a mix of alphanumeric names. The numbers cant be converted into small text with the same ratio, and it distorts the whole string.

File: src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt [hunk 6]
```diff
@@ -91,7 +91,7 @@ class MusicalMinecarts : EventMiniGame(GameConfig.MUSICAL_MINECARTS) {
 
         simpleCountdown {
             newRound()
-            Util.runAction(PlayerType.PARTICIPANT) { it.sendMessage("<game_colour>Remember, do NOT click the minecarts BEFORE the music has STOPPED... you will be stunned!".style()) }
+            Util.runAction(PlayerType.PARTICIPANT) { it.sendMessage("<game_colour>ʀᴇᴍᴇᴍʙᴇʀ, ᴅᴏ ɴᴏᴛ ᴄʟɪᴄᴋ ᴛʜᴇ ᴍɪɴᴇᴄᴀʀᴛѕ ʙᴇꜰᴏʀᴇ ᴛʜᴇ ᴍᴜѕɪᴄ ʜᴀѕ ѕᴛᴏᴘᴘᴇᴅ... ʏᴏᴜ ᴡɪʟʟ ʙᴇ ѕᴛᴜɴɴᴇᴅ!".style()) }
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt]: Bold "NOT"

File: src/main/kotlin/gg/flyte/christmas/npc/WorldNPC.kt [hunk 7]
```diff
@@ -95,9 +95,9 @@ class WorldNPC private constructor(displayName: String, textureProperties: List<
             2 to MapSinglePoint(537.3, 104.25, 499.55, -90, 0)
         )
         private val placeDefaultComponent = mapOf(
-            0 to "<colour:#ffcb1a>➊",
-            1 to "<colour:#d0d0d0>➋",
-            2 to "<colour:#a39341>➌"
+            0 to "<gold>➊",
+            1 to "<orange>➋",
+            2 to "<yellow>➌"
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/npc/WorldNPC.kt]: The team doesn't seem too enthusiastic about this colour change. It looks much more dull than what it was previously.

File: src/main/kotlin/gg/flyte/christmas/util/Colors.kt [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    val PURPLE: TextColor = TextColor.color(0x6735CE)
+    val MAGENTA: TextColor = TextColor.color(0xB119C2)
+    val LIGHT_PURPLE: TextColor = TextColor.color(0xEE57FF)
+    val RED: TextColor = TextColor.color(0xF23A3A)
+    val GOLD: TextColor = TextColor.color(0xFF9800)
+    val ORANGE: TextColor = TextColor.color(0xF19C79)
+    val PINK: TextColor = TextColor.color(0xE89EB8)
+
+    fun tagResolver(): TagResolver = TagResolver.builder().resolvers(
+        Placeholder.styling("orange", WHITE),
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/util/Colors.kt]: Is this a mistake?

File: src/main/kotlin/gg/flyte/christmas/util/Colors.kt
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/util/Colors.kt:1]: Convert the colours into a hex colour colour code. I know it's pretty evident that all you'd have to do is omit the `0x[...]`, however the rest of the plugin describes colours in a `#123abc` format, so I would do the same here.

Also... rename to `Colours.kt` please!

File: src/main/kotlin/gg/flyte/christmas/util/Extn.kt [hunk 9]
```diff
@@ -50,13 +51,13 @@ fun String.colourise(): String {
...
         replaceSharp.forEach { c ->
-            builder.append("&").append(c)
+            builder.append("§").append(c)
         }
 
         message = message.replace(hexCode, builder.toString())
     }
 
-    return ChatColor.translateAlternateColorCodes('&', message)
+    return ChatColor.translateAlternateColorCodes('§', message)
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/util/Extn.kt]: While arguably both the section sign and the ampersand symbols are legacy and outdated, various components in the plugin use ampersands rather than section signs for menu work. Ampersands are also notably easier to type out. 

I would recommend reverting this.

File: src/main/kotlin/gg/flyte/christmas/util/Extn.kt [hunk 10]
```diff
@@ -50,13 +51,13 @@ fun String.colourise(): String {
 
         val builder = StringBuilder()
         replaceSharp.forEach { c ->
-            builder.append("&").append(c)
+            builder.append("§").append(c)
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/util/Extn.kt]: See above.

File: src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt:1]: The team generally disagrees with the styling changes made here. They seem too dull and do not make the scores exciting. I would revert these changes (but keep the small-text edits), but I am considering using some darker colours.

File: src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt [hunk 11]
```diff
@@ -135,7 +135,7 @@ class ChristmasEventPlugin : JavaPlugin() {
             }
             it.billboard = Display.Billboard.CENTER
             it.isDefaultBackground = false
-            it.backgroundColor = Color.fromRGB(94, 68, 23)
+            it.backgroundColor = Color.fromARGB(0x1A000000)
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: Again, same comment with any other occurrence of `0x`.

Also, would you be able to find a new, pastel red colour for this text, and a similar pastel for the background.

File: src/main/resources/config.yml [hunk 12]
```diff
@@ -16,9 +16,11 @@ contributors:
   - "<c4ad9786-2f63-4c47-8854-6985f13ccff2><Plugin Contributor><538.5, 101, 458.5>" # kolja
   - "<4f8cc0a1-80c6-4e37-8c99-9645ad60836d><Plugin Contributor><550.5, 101, 458.5>" # penguin
   - "<9cd72c16-b422-4a48-b3ad-1541ee942b86><Resource Pack\nDesigner><536.5, 101, 459.5>" # spaceniklas
+  - "<b81d672f-a9b5-4ec9-be08-7cefb0a02505><Plugin Contributor><552.5, 101, 459.5>" # lutto
 
 # Donation Auth
 donations:
+  disabled: false
```
Comment (ShreyasAyyengar) [inline src/main/resources/config.yml]: See comment about disabled donations.

**Review by ShreyasAyyengar — CHANGES_REQUESTED** [review]
General feedback is as follows:
- Revert colours back to hex format (`#abc123`)
- Revert colours of sidebar and headboard positions.
- Tweak or remove some Extn functions.

Let me know if you have any questions.

File: src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt (hunk 3 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt:93]: This actually solves this bug, as it now parses it using the default color codes from MiniMessage instead of custom ones.

File: src/main/kotlin/gg/flyte/christmas/util/Extn.kt (hunk 9 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/util/Extn.kt]: Alright, simple change

File: src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt (hunk 6 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt]: Sure

File: src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt (hunk 5 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt]: As I do agree that it doesn't always look good, another idea might be, since you already have a resource pack, using a custom font which adds small numbers. I think that's the best idea since otherwise there is a loss in consistency in some messages.

File: src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt (hunk 4 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt:285]: Like :Component? If so, might I ask why exactly?

File: src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt (hunk 2 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: This is a very small change changing one line in the entire codebase. I see this as a small, but nice QoL feature for developers, not even meant to be used in production.

File: src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt:1]: That's fine by me, if you have any ideas yourself or the team, you can just make a change here so to not get into merging issues. Although I don't get how you want to achieve less "dullness" whilst using darker colors, I don't think I completely understand what you mean there.

File: src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt (hunk 11 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: > Also, would you be able to find a new, pastel red colour for this text, and a similar pastel for the background.

Is a pastel-like color palette also what you are aiming for? Because then I can make changes accordingly to that.

And about the actual red color, here is a palette of possible pastel-like red colors: https://artincontext.org/colorpalette/index.html?colors=DA2C38-F2545B-FF5555-FF7477-F08080-FF8B94-F4989C. **Please do not that these colors look very different when not put side-by-side like this.**

Comment (ShreyasAyyengar) [reply src/main/kotlin/gg/flyte/christmas/ChristmasEventPlugin.kt]: Screenshot 2024-12-03 at 8 11 34 am
I would probably go with a text colour like this and then a white background colour. 

Might want to do some testing with this.

File: src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt (hunk 3 above)
Comment (ShreyasAyyengar) [reply src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt:93]: > This actually solves this bug, as it now parses it using the default color codes from MiniMessage instead of custom ones.

What do your custom tag resolvers change? Are they referencing the default Minecraft chat colours, or are they slightly modified based on the palette you sent earlier? If I parsed `<red>`, does that fallback to `ChatColor.RED` or a new paletted colour?

File: src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt (hunk 4 above)
Comment (ShreyasAyyengar) [reply src/main/kotlin/gg/flyte/christmas/minigame/engine/EventController.kt:285]: Primarily for code readability and following Kotlin’s idiomatic style. In cases where the compiler can infer the return type, it’s generally considered more concise and clean to omit it unless the explicit declaration adds clarity in complex cases or is required for documentation purposes.

File: src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt (hunk 3 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/listeners/HousekeepingEventListener.kt:93]: This code uses the default MiniMessage instance, so it should only their tag resolvers. They currently do function correctly in-game.

File: src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt
Comment (ShreyasAyyengar) [reply src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt:1]: The feedback from the playtest saw that many players felt the colours were too light and were hard to see. This change overly simplifies the colours, so I am attempting to strike a balance between the pop that the colours have and readability.

File: src/main/kotlin/gg/flyte/christmas/npc/WorldNPC.kt (hunk 7 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/npc/WorldNPC.kt]: Oh, I seem to have missed some comments.
I'll change it back.

File: src/main/kotlin/gg/flyte/christmas/util/Colors.kt (hunk 8 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/util/Colors.kt]: Well this is quite embarassing...

File: src/main/kotlin/gg/flyte/christmas/util/Colors.kt
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/util/Colors.kt:1]: > Convert the colours into a hex colour colour code. I know it's pretty evident that all you'd have to do is omit the 0x[...], however the rest of the plugin describes colours in a #123abc format, so I would do the same here.

👍 

> Also... rename to Colours.kt please!

Haha! Will be sure to do so.

File: src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt (hunk 5 above)
Comment (ShreyasAyyengar) [reply src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt]: > As I do agree that it doesn't always look good, another idea might be, since you already have a resource pack, using a custom font which adds small numbers. I think that's the best idea since otherwise there is a loss in consistency in some messages.

I will see if this can be looked into. That also would mean reverting 99% of the small-text changes you have made here, but I don't see that being too difficult. For now though I would remove it. 

Fortunately, since the client already pixelates any character (small or regular), it isn't too inconsistent when a random player name is thrown into the text.

File: src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/visual/SidebarManager.kt:1]: Yeah, these changes try to keep a consistent Christmas color palette. I've reverted the changes, so you can modify anything as you wish on the master branch.

File: src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt (hunk 5 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/util/StringUtil.kt]: > I will see if this can be looked into. That also would mean reverting 99% of the small-text changes you have made here, but I don't see that being too difficult. For now though I would remove it.

Shouldn't be too difficult yeah.

> Fortunately, since the client already pixelates any character (small or regular), it isn't too inconsistent when a random player name is thrown into the text.

It's still quite noticeable and inconsistent, so a custom font might be a nice idea.

**Comment by ShreyasAyyengar** [comment]
When you're finished, nag a review via GitHub, and I'll take a look.

File: src/main/kotlin/gg/flyte/christmas/minigame/games/PaintWars.kt [hunk 13]
```diff
@@ -169,7 +169,9 @@ class PaintWars : EventMiniGame(GameConfig.PAINT_WARS) {
         scores.entries
             .sortedByDescending { it.value }
             .take(3)
-            .also { it.forEach { formattedWinners.put(it.key, "${it.value} block${if (it.value > 1) "s" else ""}") } }
+            .also { it.forEach {
+                formattedWinners[it.key] = "${it.value} ʙʟᴏᴄᴋ${if (it.value > 1) "ѕ" else ""}" }
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/games/PaintWars.kt:173]: Is this 's' here in small text? Honestly can't tell.

File: src/main/kotlin/gg/flyte/christmas/minigame/games/KingHill.kt [hunk 14]
```diff
@@ -96,7 +96,9 @@ class KingHill : EventMiniGame(GameConfig.KING_OF_THE_HILL) {
         val (first) = timeOnHill.entries
             .sortedByDescending { it.value }
             .take(3)
-            .also { it.forEach { formattedWinners.put(it.key, it.value.toString() + " second${if (it.value > 1) "s" else ""}") } }
+            .also { it.forEach { entry ->
+                formattedWinners[entry.key] = entry.value.toString() + " ѕᴇᴄᴏɴᴅ${if (entry.value > 1) "ѕ" else ""}"
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/games/KingHill.kt:103]: Is this 's' here in small text? Honestly can't tell.

File: src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt [hunk 15]
```diff
@@ -237,16 +237,16 @@ class MusicalMinecarts : EventMiniGame(GameConfig.MUSICAL_MINECARTS) {
         }
         super.eliminate(player, reason)
 
-        var value = "$roundNumber round${if (roundNumber > 1) "s" else ""}"
+        val value = "$roundNumber ʀᴏᴜɴᴅ${if (roundNumber > 1) "ѕ" else ""}"
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt:240]: Is this 's' here in small text? Honestly can't tell.

File: src/main/kotlin/gg/flyte/christmas/minigame/games/Spleef.kt [hunk 16]
```diff
@@ -157,15 +157,14 @@ class Spleef : EventMiniGame(GameConfig.SPLEEF) {
 
         super.eliminate(player, reason)
 
-        val value = "$seconds second${if (seconds > 1) "s" else ""}"
+        val value = "$seconds ѕᴇᴄᴏɴᴅ${if (seconds > 1) "ѕ" else ""}"
```
Comment (ShreyasAyyengar) [inline src/main/kotlin/gg/flyte/christmas/minigame/games/Spleef.kt:160]: Is this 's' here in small text? Honestly can't tell.

**Comment by ShreyasAyyengar** [comment]
Also, please make sure that you are reverting changes like so (mainly NPC create lines):

Screenshot 2024-12-03 at 3 54 29 pm

The section-sign must be used because the PacketEvents' NPC 'wrapper' takes a display name as a String, not a component, which means that they need to be legacy-colourised.

**Review by ShreyasAyyengar — CHANGES_REQUESTED** [review]

File: src/main/kotlin/gg/flyte/christmas/minigame/games/PaintWars.kt (hunk 13 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/minigame/games/PaintWars.kt:173]: Should be

File: src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt (hunk 15 above)
Comment (StillLutto) [reply src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt:240]: You can see it in the diff

**Review by ShreyasAyyengar — APPROVED** [review]

**Comment by ShreyasAyyengar** [comment]
[Thanks](https://github.com/flytegg/christmas-plugin-2024/commit/228edf5e8068c44ae270642fd6ec4b2a26d8c379)

Do send or PM me a Minecraft UUID so I can add you as an NPC in our contributors section as a thank you for your contribution.
